src/networks.json

```diff
@@ -1659,10 +1659,10 @@
     "network": "mainnet",
     "multicall": "0x9787792A1020B00fb65191754c3D87c89Bf36444",
     "rpc": [
-      "https://canto.neobase.one"
+      "https://canto.gravitychain.io"
```

Comment:
Hi @ColinPlatt this node is not a archive node. is there some other full archive node?

Reply:
I get this error for example
```
fullArchiveNodeStart Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":45,\"error\":{\"code\":-32000,\"message\":\"couldn't fetch balance. Node state is pruned\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x9787792a1020b00fb65191754c3d87c89bf36444\",\"0x3df194\"],\"id\":45,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://canto.gravitychain.io", code=SERVER_ERROR, version=web/5.6.0)
```